lib/Common/Common/GetCurrentFrameId.h

```diff
@@ -21,8 +21,13 @@
 #define GET_CURRENT_FRAME_ID(f) \
     __asm { mov f, ebp }
 #elif defined(_M_X64)
+#ifdef _WIN32
 #define GET_CURRENT_FRAME_ID(f) \
     (f = _ReturnAddress())
```

Comment:
`_ReturnAddress` itself has nothing wrong. I'm guessing the issue here is it cannot be used as ...FRAME_ID... Not sure of the details. Maybe because we do comparisons on it, and return address doesn't work well (e.g. we don't yet have dynamic interpreter frame) to identify a frame?